src/FixedDataTableTranslateDOMPosition.js

```diff
@@ -13,6 +13,9 @@
 import translateDOMPositionXY from 'translateDOMPositionXY';
 
 function FixedDataTableTranslateDOMPosition(/*object*/ style, /*number*/ x, /*number*/ y, /*boolean*/ initialRender = false) {
+  if (style.display === 'none') {
```

Comment:
Don't even try to translate if display is none.

But we can remove this if you think the transforms still need to be calculated. In the end, the row won't be displayed anyway.

Reply:
this is fine by me.  I don't think we need to transform (we'll see if anyone complains)